src/Compilers/Core/Desktop/CommandLine/CommonCompiler.cs

```diff
@@ -230,7 +226,6 @@ protected bool ReportErrors(IEnumerable<Diagnostic> diagnostics, TextWriter cons
             return hasErrors;
         }
 
-        protected bool ReportErrors(IEnumerable<DiagnosticInfo> diagnostics, TextWriter consoleOutput, ErrorLogger errorLogger)
```

Comment:
Strange this is being reported as a delete, I just changed the modifier to `public`